pootle/apps/accounts/utils.py

```diff
@@ -419,3 +421,34 @@ def update_user_email(user, new_email):
     validate_email(new_email)
     user.email = new_email
     user.save()
+
+
+def get_user_list_with_permission(permission_code, project, language):
```

Comment:
Yeah exactly. Also note if you make it a manager method, for consistency with manager querying methods (and developer expectations), you'll probably want to have it return a `QuerySet` instead of a list.